src/krati_Shukla/ScannerClass/CharFrequency.java

```diff
@@ -0,0 +1,39 @@
+//Find the frequency of character from a given String array.
+
+package krati_Shukla.ScannerClass;
+
+import java.util.Scanner;
+
+public class CharFrequency {
+	
+	 int count; 
```

Comment:
You should write this instance variable with in method as, it’s useful for only that method.  If variable going to use multiple places then it makes senses to define that variable at instance. just trying to explain good coding practices.